src/fontbackgroundcolor.js

```diff
@@ -140,3 +140,11 @@ export default class FontBackgroundColor extends Plugin {
  *
  * @member {Number} module:font/fontbackgroundcolor~FontBackgroundColorConfig#columns
  */
+
+/**
+ * Determines maximum number of available document colors. If not defined,
+ * then it equals to {@link module:font/fontbackgroundcolor~FontBackgroundColorConfig#columns} value.
+ * If set to `0`, then document colors feature will be disabled.
```

Comment:
```suggestion
 * Setting it to `0` will disable document colors feature.
```

Reply:
Change should also be applied to the occurrence in the font color config.

Reply:
It would be nice to add a listing with this property being used. Also it would be nice to put information about the default value as the last information (but before the example snippet).